pkg/apis/build/v1alpha1/notary_types.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+// +k8s:openapi-gen=true
+type NotaryConfig struct {
+	V1 *NotaryV1Config `json:"v1,omitempty"`
+}
+
+// +k8s:openapi-gen=true
+type NotaryV1Config struct {
+	URL             string           `json:"url"`
+	SecretRef       NotarySecretRef  `json:"secretRef"`
+	ConfigMapKeyRef *ConfigMapKeyRef `json:"configMapKeyRef,omitempty"`
```

Comment:
Is `configMapKeyRef` common for config map references in the k8s api?

Reply:
I borrowed it from the pod spec.